src/components/card/card.scss

```diff
@@ -0,0 +1,196 @@
+@import "shadows";
+@import "default-theme";   // TODO: Remove this
+
+$md-card-default-padding: rem(2.400) !default;
+$md-card-mobile-padding: rem(2.400) rem(1.600) !default;
+$md-card-border-radius: 2px !default;
+$md-card-header-size: rem(4.000) !default;
+
+md-card {
+  display: block;
+  position:relative;
+  padding: $md-card-default-padding;
+  border-radius: $md-card-border-radius;
+  box-shadow: $md-shadow-bottom-z-1;
+  font-family: $font-family;
+  background: md-color($md-background, 50);       // temp until 0 works
+}
+
+%md_card_section_base {
+  display: block;
+  margin-bottom: rem(1.600);  
+}
+
+md-card-title {
+  @extend %md_card_section_base;
+  font-size: rem(2.400);
+  font-weight: 400;
+}
+
+md-card-subtitle {
+  @extend %md_card_section_base;
+  font-size: $md-body-font-size-base;
+  color: md-color($md-foreground, "secondary-text");
+}
+
+md-card-content {
+  @extend %md_card_section_base;
+  font-size: $md-body-font-size-base;
+}
+
+md-card-actions {
+  @extend %md_card_section_base;
+  margin-left: rem(-1.600);
+  margin-right: rem(-1.600);
+  padding: rem(0.800) 0;
+}
+
+img[md-card-image] {
+  width: calc(100% + 48px);
+  margin: 0 rem(-2.400) rem(1.600) rem(-2.400);
+}
+
+img[md-card-xl-image] {
+  width: rem(24.00);
+  height: rem(24.00);
+  margin: rem(-0.800);
+}
+
+md-card-footer {
+  position: absolute;
+  bottom: 0;
+}
+
+md-card-actions {
+  [md-button], [md-raised-button], button {
+    margin: 0 rem(0.400);
+  } 
+}
+
+/* HEADER STYLES */
+
+md-card-header {
+  display: flex;
+  flex-direction: row;
+  height: $md-card-header-size;
+  margin: rem(-0.800) 0 rem(1.600) 0;
+}
+
+.md-card-header-text {
+  height: $md-card-header-size;
+  margin: 0 rem(0.800);
+}
+
+img[md-card-avatar] {
+  height: $md-card-header-size;
+  width: $md-card-header-size;
+  border-radius: 50%;
+}
+
+md-card-header md-card-title {
+  font-size: $md-body-font-size-base;
+}
+
+/* TITLE-GROUP STYLES */
+
+%md_card_title_img {
+  margin: rem(-0.800) 0;
+}
+
+md-card-title-group {
+  display: flex;
+  justify-content: space-between;
+  margin: 0 rem(-0.800);
+}
+
+img[md-card-sm-image] {
+  @extend %md_card_title_img;
+  width: rem(8.00);
+  height: rem(8.00);
+}
+
+img[md-card-md-image] {
+  @extend %md_card_title_img;
+  width: rem(11.200);
+  height: rem(11.200);
+}
+
+img[md-card-lg-image] {
+  @extend %md_card_title_img;
+  width: rem(15.200);
+  height: rem(15.200);
+}
+
+/* MEDIA QUERIES */
+
+@media (max-width: $md-mobile-screen){
+  md-card {
+    padding: $md-card-mobile-padding;
+  }
+
+  img[md-card-image] {
+    width: calc(100% + 32px);
+    margin: rem(1.600) rem(-1.600);
+  }
+
+  md-card-title-group {
+    margin: 0;
+  }
+
+  img[md-card-xl-image] {
+    margin-left: 0;
+    margin-right: 0;
+  }
+
+  md-card-header {
+    margin: rem(-0.800) 0 0 0;
+  }
+
+}
+
+/* FIRST/LAST CHILD ADJUSTMENTS */
+
+img[md-card-image]:first-child {
+  margin-top: rem(-2.400);
+}
+
+md-card-header md-card-subtitle:not(:first-child) {
```

Comment:
Instead of doing `:first-child` and then `:not(:first-child)`, why not just define the non-first-child style first and then override it with the `:first-child` ?